### PR TITLE
VideoPreviewV1

### PR DESCRIPTION
**Pull request by zeitounmax**

### CODE REVIEW

File: frontend/src/components/VideoPreview.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function VideoPreview({ video }) {
+  return (
+    <div className="video-preview">
```
Comment (Justine-Merlin) [inline frontend/src/components/VideoPreview.jsx]: This `div` seems to be a `figure` :eyes: [Documentation](https://developer.mozilla.org/fr/docs/Web/HTML/Element/figure)

**Review by Justine-Merlin — CHANGES_REQUESTED** [review]

File: frontend/src/components/VideoPreview.jsx (hunk 1 above)
Comment (ghost) [reply frontend/src/components/VideoPreview.jsx]: Resolve Div to Figure  in progress

Comment (rocambille) [reply frontend/src/components/VideoPreview.jsx]: figure should have a figcaption

File: frontend/src/components/VideoPreview.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function VideoPreview({ video }) {
+  return (
+    <figure className="video-preview">
+      <img src={video.thumbnail} alt={video.title} />
+      <div className="video-info">
+        <h3>{video.title}</h3>
+        <figcaption>{video.description}</figcaption>
```
Comment (rocambille) [inline frontend/src/components/VideoPreview.jsx]: figcaption should be a direct child of figure : your div shoudl be a figcaption. the description should be a paragraph

image

**Review by rocambille — CHANGES_REQUESTED** [review]

**Pull request merged**